### DISCUSSION TITLE
Wrong serial number on label of HM-600

### DISCUSSION THREAD

**Discussion by schnoog** [tbnobody/OpenDTU#1145] [General]
Hi,
thanks for this great piece of software.

I lately bought a BKW with a HM-600. 
When setting it up in OpenDTU I've entered the serial printed on the label and failed to get any response. 
I just stumbled over the wiki page which explains the serial number structure and was quite surprised.
My SN is 1141190169XXX (really, I checked it twice, and both labels on the HM have the same number).
Since the year-index 1 (5. char) doesn't make any sense, i tried 114190169XXX instead. 
And suddenly it works. 

So if anybody also struggles with the HM-600 connection and also has 114119 serial number: Don't give up and try 11419 instead. 

Edit: Sorry for the confusion, I messed the numbers up in the initial post and just corrected it 
Update: My OH (better eyes than me, just mentioned that the  11 after 114 looks like 1I  (the second "1" has only the vertical line), so seems like a print problem.

**Comment by schnoog** [comment]
Good morning.

It seems there's still a problem.

> Fetch inverter: 114190169XXX
> 08:46:35.388 > Request SystemConfigPara
> 08:46:35.452 > TX RealTimeRunData Channel: 23 --> 15 90 16 98 95 80 11 11 68 80 0B 00 64 B2 40 C9 00 00 00 00 00 00 00 00 7E B5 69 
> 08:46:35.980 > RX Period End
> 08:46:35.980 > All missing
> 08:46:35.980 > Nothing received, resend whole request
> 08:46:35.980 > TX RealTimeRunData Channel: 40 --> 15 90 16 98 95 80 11 11 68 80 0B 00 64 B2 40 C9 00 00 00 00 00 00 00 00 7E B5 69 
> 08:46:36.524 > RX Period End
> 08:46:36.524 > All missing
> 08:46:36.524 > Nothing received, resend whole request
> 08:46:36.524 > TX RealTimeRunData Channel: 61 --> 15 90 16 98 95 80 11 11 68 80 0B 00 64 B2 40 C9 00 00 00 00 00 00 00 00 7E B5 69 
> 08:46:37.072 > RX Period End
> 08:46:37.072 > All missing
> 08:46:37.072 > Nothing received, resend whole request
> 08:46:37.072 > TX RealTimeRunData Channel: 75 --> 15 90 16 98 95 80 11 11 68 80 0B 00 64 B2 40 C9 00 00 00 00 00 00 00 00 7E B5 69 
> 08:46:37.619 > RX Period End
> 08:46:37.619 > All missing
> 08:46:37.619 > Nothing received, resend whole request
> 08:46:37.619 > TX RealTimeRunData Channel: 3 --> 15 90 16 98 95 80 11 11 68 80 0B 00 64 B2 40 C9 00 00 00 00 00 00 00 00 7E B5 69 
> 08:46:38.117 > RX Period End
> 08:46:38.117 > All missing
> 08:46:38.117 > Nothing received, resend count exeeded
> 08:46:38.250 > TX AlarmData Channel: 23 --> 15 90 16 98 95 80 11 11 68 80 11 00 64 B2 40 C9 00 00 00 00 00 00 00 00 A4 AE B2 
> 08:46:38.964 > RX Period End
> 08:46:38.964 > All missing
> 08:46:38.964 > Nothing received, resend whole request
> 08:46:38.964 > TX AlarmData Channel: 40 --> 15 90 16 98 95 80 11 11 68 80 11 00 64 B2 40 C9 00 00 00 00 00 00 00 00 A4 AE B2 
> 08:46:39.751 > RX Period End
> 08:46:39.751 > All missing
> 08:46:39.751 > Nothing received, resend whole request
> 08:46:39.751 > TX AlarmData Channel: 61 --> 15 90 16 98 95 80 11 11 68 80 11 00 64 B2 40 C9 00 00 00 00 00 00 00 00 A4 AE B2 
> 08:46:40.550 > RX Period End
> 08:46:40.550 > All missing
> 08:46:40.550 > Nothing received, resend whole request
> 08:46:40.550 > TX AlarmData Channel: 75 --> 15 90 16 98 95 80 11 11 68 80 11 00 64 B2 40 C9 00 00 00 00 00 00 00 00 A4 AE B2 
> 08:46:41.342 > RX Period End
> 08:46:41.342 > All missing
> 08:46:41.342 > Nothing received, resend whole request
> 08:46:41.342 > TX AlarmData Channel: 3 --> 15 90 16 98 95 80 11 11 68 80 11 00 64 B2 40 C9 00 00 00 00 00 00 00 00 A4 AE B2 
> 08:46:42.139 > RX Period End
> 08:46:42.139 > All missing
> 08:46:42.139 > Nothing received, resend count exeeded
> 08:46:42.204 > TX SystemConfigPara Channel: 23 --> 15 90 16 98 95 80 11 11 68 80 05 00 64 B2 40 C9 00 00 00 00 00 00 00 00 B0 BA A6 
> 08:46:42.437 > RX Period End
> 08:46:42.437 > All missing
> 08:46:42.437 > Nothing received, resend whole request
> 08:46:42.437 > TX SystemConfigPara Channel: 40 --> 15 90 16 98 95 80 11 11 68 80 05 00 64 B2 40 C9 00 00 00 00 00 00 00 00 B0 BA A6 
> 08:46:42.676 > RX Period End
> 08:46:42.676 > All missing
> 08:46:42.676 > Nothing received, resend whole request
> 08:46:42.676 > TX SystemConfigPara Channel: 61 --> 15 90 16 98 95 80 11 11 68 80 05 00 64 B2 40 C9 00 00 00 00 00 00 00 00 B0 BA A6 
> 08:46:42.922 > RX Period End
> 08:46:42.922 > All missing
> 08:46:42.922 > Nothing received, resend whole request
> 08:46:42.922 > TX SystemConfigPara Channel: 75 --> 15 90 16 98 95 80 11 11 68 80 05 00 64 B2 40 C9 00 00 00 00 00 00 00 00 B0 BA A6 
> 08:46:43.167 > RX Period End
> 08:46:43.167 > All missing
> 08:46:43.167 > Nothing received, resend whole request
> 08:46:43.167 > TX SystemConfigPara Channel: 3 --> 15 90 16 98 95 80 11 11 68 80 05 00 64 B2 40 C9 00 00 00 00 00 00 00 00 B0 BA A6 
> 08:46:43.375 > RX Period End
> 08:46:43.375 > All missing
> 08:46:43.375 > Nothing received, resend count exeeded
> 08:46:43.375 > Fetch inverter: 114190169XXX
> 08:46:43.375 > Request SystemConfigPara
> 08:46:43.477 > TX RealTimeRunData Channel: 23 --> 15 90 16 98 95 80 11 11 68 80 0B 00 64 B2 40 D1 00 00 00 00 00 00 00 00 7E 1F DB 
> 

The inverter is correctly identified  (as HM-600, HM-700, HM-800) when I use the serial without said "1".
If I enter the full serial it remains "unknown".
I tried all power settings (also added a 100uF cap)

**Comment by tbnobody** [comment]
Not sure how you changed your inverters serial number but 
```
Label:   1141190189XXX
OpenDTU: 1141 90169XXX
```

I would have expected that you just change the year number but you've also exchanged the 8 by a 6. (especially the last part of the serial number is important for the communication whereby the first part is used for identification of the inverter)

**Reply by schnoog** [reply]
Thank you very much for your resposne.
With changed I meant in the OpenDTU GUI, not the inverter itself.
From 
1141190169XXX
to
114190169XXX
So simply left out the 1.
(both 169XXX, messed that up in my first post, sorry)


My OH just mentioned that the 11 looks more like 1I ...
Seems like the labels are misprinted. Barcodes are also not working.
So I guess the 11419... is the correct serial and the label printer messed it up.

**Reply by schnoog** [reply]
Pre-lunchtime-update:

I deleted the inverter from OpenDTU, pulled the plug out the Inverter, plugged it in again, added the 114190169XXX serial
and it works. I receive the data from the HM-600.

Thanks again for your effort and this great piece of software.
